### ISSUE TITLE
Error when using --config musejazz on macOS

### ISSUE THREAD

**Issue by simonkeys** [ChordPro/chordpro#136]
I was excited to see the possibility of using the MuseJazz font as described here: https://groups.io/g/ChordPro/topic/advanced_chord_formatting/72673346

Using version 0.977  and `chordpro --config musejazz test.cho` I got the error `musejazz: No such file or directory`.

After installing the dev version 0.977_038 and putting the MuseJazz fonts in the `.fonts` folder, with 

```perl script/chordpro --config musejazz test.cho``` 

I got the error 

`Can't call method "isvirtual" on an undefined value at /Users/simon/perl5/perlbrew/perls/perl-5.32.1/lib/site_perl/5.32.1/PDF/API2/Content.pm line 1578.`

I tried both the system perl and a current version of perl and got the same error. Downgrading PDF::API2 to an older version also had no effect.

My perl chops are extremely rusty, but I can try to help with debugging if you need someone on macOS.

I'm using the test file

```
{title: By Way Of Sorrow}

[G#m]You've been taken [F#]by the [B]wind
You have [E]known the [B]kiss of [F#]sor[E]row
[G#m]Doors that would not [F#]take you [B]in
[E]Outcast [B]and a [F#]stran[E]ger
```

 - OS: macOS 10.14.6 (18G7016)
 - Perl version 5.18.4 built for darwin-thread-multi-2level (also tried 5.32.1)
 - ChordPro version 0.977_038
 - Where did you get ChordPro from: GitHub

**Comment by sciurius** [comment]
The "isvirtual" error is a weird way of telling you that it cannot find the desired font.

The required file is `~/.fonts/MuseJazz.ttf`. Can you double check that it is there, exactly spelled like that?

**Comment by simonkeys** [comment]
I see, thanks. The MuseJazz fonts I found are actually otf fonts: https://github.com/musescore/MuseScore/tree/master/fonts/musejazz

I edited the musejazz.json file to load the MuseJazz.otf file (also tried MuseJazzText.otf), which it could do, but these fonts apparently don't match well enough, since I got some blank squares instead of characters.

Now that I see how it's done, though, I think I'm going to just edit the config file to work with my jazz font of choice. So thanks for getting me going again!
